### ISSUE TITLE
Remove generation of structure files from CONTENTdm compound toolchain

### ISSUE THREAD

**Issue by mjordan** [MarcusBarnes/mik#249]
The Islandora Compound Batch module contains a utility to generate these files. Discussion at https://github.com/MarcusBarnes/islandora_compound_batch/issues/2 led to the decision to remove equivalent functionality from MIK.

**Comment by mjordan** [comment]
@MarcusBarnes, output of the CdmCompound toolchain now looks like:

```
.
├── 1971
│   ├── 1969
│   │   ├── MODS.xml
│   │   └── OBJ.jp2
│   ├── 1970
│   │   ├── MODS.xml
│   │   └── OBJ.jp2
│   └── MODS.xml
├── 1974
│   ├── 1972
│   │   ├── MODS.xml
│   │   └── OBJ.jp2
│   ├── 1973
│   │   ├── MODS.xml
│   │   └── OBJ.jp2
│   └── MODS.xml
├── 1977
│   ├── 1975
│   │   ├── MODS.xml
│   │   └── OBJ.jp2
│   ├── 1976
│   │   ├── MODS.xml
│   │   └── OBJ.jp2
│   └── MODS.xml
├── mik.log
└── problem_records.log
```

I see no need to test further if the output looks correct. Your call.

**Comment by MarcusBarnes** [comment]
@mjordan  Thanks. If this is ready, please create a pull-request.

**Comment by mjordan** [comment]
PR is at https://github.com/MarcusBarnes/mik/pull/250

**Comment by MarcusBarnes** [comment]
Resolved with 9c28054.
